os_info.yaml

```diff
@@ -136,3 +136,9 @@ ubuntu-2210:
     - amd64
     - aarch64
   type: deb
+ubuntu-2304:
+  version_name: luna
+  arch:
+    - amd64
+    - aarch64
```

Comment:
do we no longer want ppc64le for ubuntu?

Reply:
We're keeping ppc64le and s390x for LTS ubuntu releases only, mainly just a resource limitation but also people using those tend to stick with LTS only.